content/docs/06-integrations/00-minio-operator.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'MinIO Operator'
+metaTitle: 'MinIO Operator'
+metaDescription: 'MinIO Operator Distributed Object Storage pack in Spectro Cloud'
+hiddenFromNav: true
+isIntegration: true
+hideToC: false
+category: ['storage']
+logoUrl: 'https://registry.spectrocloud.com/v1/palette-upgrader/blobs/sha256:b6081bca439eeb01a8d43b3cb6895df4c088f80af978856ddc0da568e5c09365?type=image/png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# MinIO Operator Overview
+
+MinIO is a Kubernetes-native high performance object store with an Amazon S3-compatible API. Use MinIO to build high performance infrastructure for machine learning, analytics, and application data workloads.
```

Comment:
`MinIO is a Kubernetes-native high-performance object store with an Amazon S3-compatible API. Use MinIO to build high-performance infrastructure for machine learning, analytics, and application data workloads.`